web/packages/teleport/src/Notifications/Notifications.tsx

```diff
@@ -225,7 +232,9 @@ export function Notifications({ iconSize = 24 }: { iconSize?: number }) {
 
       <NotificationsDropdown
         open={open}
+        id="tb-notifications-dropdown"
```

Comment:
Could you extract those IDs to variables outside of the component body so to guarantee that `id` and `aria-controls` always match?